syncd/scripts/syncd_init_common.sh

```diff
@@ -27,9 +27,8 @@ fi
 # Use temporary view between init and apply
 CMD_ARGS+=" -u"
 
-# Use bulk api`s in SAI
-# currently disabled since most vendors don't support that yet
-# CMD_ARGS+=" -l"
+# Use bulk APIs in SAI
```

Comment:
I think we want to add this flag just for SODA and not for everbody given that comment. Update the `CMD_ARGS` inside the `config_syncd_soda()` function so it'll be applied only for SODA builds.

Reply:
OK - I've modified - please rereview